client/my-sites/email/email-forwarding/email-forwarding-add-new.jsx

```diff
@@ -142,18 +151,19 @@ const EmailForwardingAddNew = createReactClass( {
 				{ this.cancelButton() }
 			</FormFooter>
 		);
-	},
+	}
 
 	formFields() {
 		if ( ! this.shouldShowForm() ) {
 			return null;
 		}
 
-		const contactText = this.props.translate( 'contact', {
+		const { translate, selectedDomainName } = this.props,
+			contactText = translate( 'contact', {
```

Comment:
I prefer to define the variables individually:
```
const var1 = 'foo1';
const var2= 'foo2';

Reply:
👍